src/main/kotlin/org/jitsi/nlj/dtls/DtlsStack.kt

```diff
@@ -196,7 +197,7 @@ class DtlsStack(
     override fun receive(buf: ByteArray, off: Int, length: Int, waitMillis: Int): Int {
         val packetInfo = incomingProtocolData.poll(waitMillis.toLong(), TimeUnit.MILLISECONDS) ?: return -1
         val packet = packetInfo.packet
-        System.arraycopy(packet.buffer, 0, buf, off, Math.min(length, packet.length))
+        System.arraycopy(packet.buffer, packet.offset, buf, off, min(length, packet.length))
```

Comment:
should we warn if the length is greater than packet.length, or it doesn't matter?

Reply:
I'll just use `packet.length`, as that's what the length is. It would be an invalid packet if it's length spanned more than the buffer's length and in other code we don't handle this condition explicitly (we'll see an exception if it happens which is fine).